README.md

```diff
@@ -1 +1 @@
-#  REVIEW PAST PROJECT 
\ No newline at end of file
+# Survey Form
```

Comment:
Try to use clear comments, your comment above is a little vague, to make it clear, the student should be shown the importance of a good README file
Example: Your README file is currently empty. Why don't you add content, project description, listing the technology used in the project and how to run it, to them so your README is more informative and professional? It would be better as required in this project to use this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.